build/target-repository/e2e/parse-php7-code/composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "autoload": {
         "psr-4": {
-            "Foo\\": "src"
+            "Foo7\\": "src"
```

Comment:
e2e test is run independently, so duplicated namespace is fine. it executed inside of per-e2e dir:

https://github.com/rectorphp/rector/blob/7c6e8362953e79ed92b439fd823595eb3c5ec09f/.github/workflows/e2e.yaml#L20-L40